batch/sql/dedup_billing_project_users_by_date.py

```diff
@@ -158,12 +158,13 @@ async def audit_changes(db):
   LOCK IN SHARE MODE
 ) AS old
 LEFT JOIN (
-  SELECT billing_date, billing_project, `user`, resource_id, CAST(COALESCE(SUM(`usage`), 0) AS SIGNED) AS `usage`
+  SELECT billing_date, billing_project, `user`, deduped_resource_id, CAST(COALESCE(SUM(`usage`), 0) AS SIGNED) AS `usage`
   FROM aggregated_billing_project_user_resources_by_date_v3
+  LEFT JOIN resources ON resources.resource_id = aggregated_billing_project_user_resources_by_date_v3.resource_id
```

Comment:
Just double-checking, this LEFT JOIN is intentionally resource_id not deduped?

Reply:
I'm pretty sure this is correct. The `v3` table doesn't have a `deduped_resource_id` to even join on.